lib/ddtrace/opentracer.rb

```diff
@@ -1,7 +1,6 @@
 # typed: strict
 require 'opentracing'
 require 'opentracing/carrier'
-require 'ddtrace'
 require 'ddtrace/opentracer/carrier'
```

Comment:
Similar to my previous comments, this file does not get required when doing `require 'ddtrace'`, it's added explicitly when following https://docs.datadoghq.com/tracing/setup_overview/setup/ruby/#quickstart-for-opentracing .

(Although the docs there are a bit redundant, since they tell people to add a `require 'ddtrace'` + `require 'opentracing'` which ddtrace does anyway)